src/main/kotlin/org/rekotlin/BlockSubscriber.kt

```diff
@@ -0,0 +1,16 @@
+package org.rekotlin
+
+/**
+ * BlockSubscriber allows subscription to a single state. This methodology is useful when subscribing multiple states
+ * within single class.
+ *
+ * Ex
+ * val subscriber : BlockSubscriber<MyState>= BlockSubscriber { myState -> }
+ * store.subscribe(subscriber)
+ */
+class BlockSubscriber<S>(private val block: (S) -> Unit) : StoreSubscriber<S> {
```

Comment:
Also please update the test cases or add new test cases for testing subscription multiple states in a single class

Reply:
Added test case